sdk/storage/tests.yml

```diff
@@ -1,30 +1,24 @@
 trigger: none
 
 extends:
-  template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+  template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
```

Comment:
Please leave this as the absolute path so that we don't have to do anything if we ever move this file around.